frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java

```diff
@@ -35,6 +39,8 @@
       IExpectsKeyProject,
       ISyncToRemoteStatusDependentAction {
 
+  public static final String RESET_INFO_SHOWN = "git-machete.reset.info.shown";
```

Comment:
Hmm is it the convention for `PropertiesComponent` keys? For instance the VCS related ones have UPPER_CASE:

```js
    propertiesComponent.setValue('ASKED_ADD_EXTERNAL_FILES', true);
    propertiesComponent.setValue('ASKED_SHARE_PROJECT_CONFIGURATION_FILES', true);
```

Reply:
there is no convention XD 
```
REQUEST_FEEDBACK = "request-lint-feedback";
// and in the other place
String PREFERENCE_KEY_PREFIX = "ScenePreference";
String SHOW_TOOLTIP_KEY = PREFERENCE_KEY_PREFIX + "ShowToolTip";
```

But we can use the VCS version as it is closer to us

Reply:
Heh okay ;D use your judgement

Reply:
It is not consistent even within VCS... :c leaving as it is